projects/packages/assets/src/class-assets.php

```diff
@@ -407,7 +408,7 @@ function ( $d ) {
 			);
 		}
 
-		if ( $options['css_path'] && file_exists( "$dir/{$options['css_path']}" ) ) {
+		if ( is_string( $options['css_path'] ) && $options['css_path'] !== '' && file_exists( "$dir/{$options['css_path']}" ) ) {
```

Comment:
Is `css_path` is protected against traversal attacks? This makes me a bit nervous. 😬

Reply:
Fair. The danger is if somewhere up the chain we allow user input. But it seems rather unlikely in this scenario, and we're restricting to `.css` files.